src/main/java/io/github/cottonmc/energy/api/ElectricalEnergyType.java

```diff
@@ -1,20 +1,35 @@
 package io.github.cottonmc.energy.api;
 
+import net.minecraft.text.TextComponent;
+import net.minecraft.text.TranslatableTextComponent;
+
 public class ElectricalEnergyType implements EnergyType {
```

Comment:
I don't think having one Type for all Electrical Energy Types is a good idea. After all, we are using different types to check whether or not a machine can accept a packet.
What about using a EnergyPacketBuilder instead that builds packets with different Types depending on the given amount. And if all energy types use the same scale (100 Low Voltage = 100 High Voltage) but have different min values we can even compare them.

Reply:
I'm having a bit of a hard time understanding this. Each current packet is an instance of ElectricalEnergyType, which implements EnergyType. All the current energy types are different types, that use the same logic but a different parameter for construction.

Reply:
Ohhh, I might have misunderstood something. give me a moment.